### PR TITLE
[vtctld] sleep/ping tablets

### PR DESCRIPTION
**Pull request by ajm188**
## Description

This PR migrates the `Sleep` and `Ping` legacy vtctl commands, renamed to `SleepTablet` and `PingTablet` since that is the component they operate on.

### Demo

```
❯ vtctldclient --server ":15999" SleepTablet zone1-100 10ms
❯ time vtctldclient --server ":15999" SleepTablet zone1-100 10s
vtctldclient --server ":15999" SleepTablet zone1-100 10s  0.01s user 0.01s system 0% cpu 10.032 total
❯ time vtctldclient --server ":15999" SleepTablet zone1-100 10ms
vtctldclient --server ":15999" SleepTablet zone1-100 10ms  0.01s user 0.01s system 59% cpu 0.040 total
❯ time vtctldclient --server ":15999" SleepTablet zone1-100 1s
vtctldclient --server ":15999" SleepTablet zone1-100 1s  0.02s user 0.01s system 2% cpu 1.035 total
❯ vtctldclient --server ":15999" PingTablet zone1-100
```

## Related Issue(s)

#7135 

## Checklist
- [x] Should this PR be backported? **No**
- [x] Tests were added or are not required
- [x] Documentation was added or is not required

### CODE REVIEW

**Comment by ajm188** [comment]
```
Run gofmt -l . | grep -vF vendor/ && exit 1 || echo "All files formatted correctly"
vitess-mixin/tools.go
Error: Process completed with exit code 1.
```

File: go/vt/vtctl/grpcvtctldserver/server.go [hunk 1]
```diff
@@ -1925,6 +1945,35 @@ func (s *VtctldServer) ShardReplicationPositions(ctx context.Context, req *vtctl
...
+	span, ctx := trace.NewSpan(ctx, "VtctldServer.SleepTablet")
+	defer span.Finish()
+
+	span.Annotate("tablet_alias", topoproto.TabletAliasString(req.TabletAlias))
+
+	dur, ok, err := protoutil.DurationFromProto(req.Duration)
+	if err != nil {
+		return nil, err
+	} else if !ok {
+		dur = *topo.RemoteOperationTimeout
```
Comment (doeg) [inline go/vt/vtctl/grpcvtctldserver/server.go:1959]: I'm kinda surprised that we don't return if we can't parse the duration. Does using `RemoteOperationTimeout` here mean "sleep for the maximum timeout"? Or will `tmc.Sleep` fail? 

(I did a bit of hunting in the codebase to answer this for myself but wound up in an interface declaration, and `Sleep` is... a cumbersome search term, haha.)

**Review by doeg — COMMENTED** [review]
This looks good as always. I have two small questions.

File: go/vt/vtctl/grpcvtctldserver/server.go (hunk 1 above)
Comment (ajm188) [reply go/vt/vtctl/grpcvtctldserver/server.go:1959]: the three-tuple here is `(parsedDuration, wasDurationSet, error)`, so if the duration is `nil` you get back `0, false, nil`, and we default to `*topo.RemoteOperationTimeout`. If we actually fail to parse, then the third value is non-nil and we take the earlier case (`err != nil`)

Comment (ajm188) [reply go/vt/vtctl/grpcvtctldserver/server.go:1959]: (it's named "ok" to mirror the map two-value semantic of (value, isThereAValue))

File: go/cmd/vtctldclient/internal/command/tablets.go [hunk 2]
```diff
@@ -105,6 +115,14 @@ Valid output formats are "awk" and "json".`,
 		Args:                  cobra.ExactArgs(2),
 		RunE:                  commandSetWritable,
 	}
+	// SleepTablet makes a SleepTablet gRPC call to a vtctld.
+	SleepTablet = &cobra.Command{
+		Use:                   "SleepTablet <alias> <duration>",
+		Short:                 "Blocks the action queue on the specified tablet for the specified amount of time. This is typically used for testing.",
```
[the remark this answers is not in the document]
Comment (ajm188) [reply go/cmd/vtctldclient/internal/command/tablets.go:121]: will do!!

**Review by deepthi — APPROVED** [review]

**Pull request merged**